_includes/language.md

```diff
@@ -0,0 +1,28 @@
+{%- assign payment_instrument = include.payment_instrument -%}
+{%- assign payment_instrument_title = payment_instrument | capitalize -%}
+{%- assign language_codes = "en-US, nb-NO, sv-SE" -%}
+
+## Languages
```

Comment:
I would like to see a correlation between the header this file puts out, and the name of the file.